optuna/integration/wandb.py

```diff
@@ -72,6 +75,11 @@ def objective(trial):
             Set of arguments passed when initializing Weights & Biases run.
             Please refer to `Weights & Biases API documentation
             <https://docs.wandb.ai/ref/python/init>`_ for more details.
+        as_multirun:
+            Creates new runs for each trial. Useful for generating W&B Sweeps like
+            panels (for ex., parameter importance, parallel coordinates, etc).
+        multirun_tag:
+            Tag name to tag runs which are generated during the study.
```

Comment:
Do you plan to implement this argument since only `as_multirun` has been added to the arguments of the callback?

Reply:
Nope. I added this initially, and forgot to update. I’ll fix these comment sections.

Reply:
Thank you for your clarification.